### PR TITLE
Return non-zero on matches

### LINKED ISSUE #3: Return non-zero on issues found
Would it make sense for this tool to return non-zero on failure or provide a flag to do this? 

This would allow for easily integration on CI to fail a build for example if something is found.

### PR DESCRIPTION
**Pull request by inverse**
Resolves #3

### CODE REVIEW

File: tell_me_your_secrets/__main__.py [hunk 1]
```diff
@@ -35,6 +35,7 @@
 arguement_parser.add_argument('-c','--config',help='Path To Another config.yml for Extracting The Data')
 arguement_parser.add_argument('-w','--write',help='Path of the csv File to which results are written')
 arguement_parser.add_argument('-f','--filter',help='Filter the Signatures you want to apply. ',nargs='+')
+arguement_parser.add_argument('-e','--exit',help='Exit non-zero on results found. ',action='store_true')
```
Comment (inverse) [inline tell_me_your_secrets/__main__.py:38]: Not sure on the best naming for this - open to suggestions.

**Comment by valayDave** [comment]
Referenced in v1.76

**Comment by inverse** [comment]
@valayDave  How comes this one was closed? This was solving the returning non-zero or did I miss something?

File: tell_me_your_secrets/__main__.py (hunk 1 above)
Comment (valayDave) [reply tell_me_your_secrets/__main__.py:38]: Refactored this code into v1.76. I can reopen if required.

Comment (inverse) [reply tell_me_your_secrets/__main__.py:38]: What do you mean by that? Ahh I can see the changes in https://github.com/valayDave/tell-me-your-secrets/commit/c116019fd38dcc4d2fe3f48f2b873aff52b40230

However can't see them on the `master` branch?

Comment (inverse) [reply tell_me_your_secrets/__main__.py:38]: I see it now - wonder how I missed it :/

**Pull request closed without merging**